ci/pipeline.yml

```diff
@@ -184,8 +184,10 @@ jobs:
       trigger: true
     - get: pipeline-tasks
     - get: broker-release
+      passed: [deploy-staging]
```

Comment:
I think that would be much better, but when I plugged that in instead of deploy-staging, it said:
```
	jobs.deploy-production.plan[0].aggregate[2].get.broker-release.passed references a job ('acceptance-tests-staging') which doesn't interact with the resource ('broker-release')
	jobs.deploy-production.plan[0].aggregate[3].get.stemcell.passed references a job ('acceptance-tests-staging') which doesn't interact with the resource ('stemcell')
```
Do you think we should add the stemcell and release to the acceptance test to solve this?

Reply:
That's a pattern we've used in other pipelines: https://github.com/18F/cg-deploy-cf/blob/master/ci/pipeline.yml#L567-L582.